deploy/layered-sre-authorization/01-layered-cs-sre-admin-project.ClusterRole.yaml

```diff
@@ -46,4 +46,21 @@ rules:
   resources:
   - '*'
   verbs:
-  - '*'
\ No newline at end of file
+  - '*'
+# CS SRE can interact with secrets, configmaps and PVC's
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  - configmaps
+  - persistentvolumeclaims
+  verbs:
+  - "*"
+# CS SRE can interact with installplans
+- apiGroups:
+  - "operators.coreos.com"
+  resources:
+  - installplans
+  verbs:
+  - "patch"
```

Comment:
What is CS SRE patching in InstallPlans?  I expect admins to interact w/ operators via subscriptions.  Maybe cleanup things w/ delete CSV.

Reply:
We mark the subscription as approved to begin the installation.